healthcheck/client.go

```diff
@@ -0,0 +1,61 @@
+package healthcheck
+
+import (
+	"github.com/ONSdigital/go-ns/rhttp"
+	"net/http"
+	"fmt"
+)
+
+// healthcheckClient is an implementation of Client that can be used to call the healthcheck endpoint of any service
+type healthcheckClient struct {
+	Client  httpClient
+	Url     string
+	Service string
+}
+
+// httpClient has a get method. Implemented by http.Client etc.
+type httpClient interface {
+	Get(url string) (*http.Response, error)
+}
```

Comment:
It wasn't feeling right that this interface would be private - it's because I think it should be a parameter in the constructor function. For that reason I would make it public and have the mock generation directive in this file for completeness. (sorry I did not spot this earlier)